contracts/wrapper/NameWrapper.sol

```diff
@@ -573,8 +584,12 @@ contract NameWrapper is
         pure
         returns (bytes memory ret)
     {
-        require(bytes(label).length > 0, "NameWrapper: Label too short");
-        require(bytes(label).length < 256, "NameWrapper: Label too long");
+        if (bytes(label).length < 1) {
+            revert LabelTooShort(label);
```

Comment:
`label` here will always be the empty string, so I think we can probably leave it out as an argument.